content/en/monitors/guide/suppress-alert-with-downtimes.md

```diff
@@ -102,65 +115,151 @@ Open the [manage Downtime page][1] and add a new downtime. Select `recurring`:
 
 Using the same example, you may want to also mute this service during the weekdays outside of business hours.
 
+{{< tabs >}}
+{{% tab "API " %}}
+
 With the following API call, you can mute alerts every weekday from 8pm to 6am:
 
+```bash
+curl -X POST "https://api.<DATADOG_SITE>/api/v1/downtime" \
+-H "Content-type: application/json" \
+-H "DD-API-KEY: ${api_key}" \
+-H "DD-APPLICATION-KEY: ${app_key}" \
+-d '{"data":{"type":"downtime","attributes":{"monitor_identifier":{"monitor_tags":["*"]},"scope":"env:prod","display_timezone":"Europe/Berlin","message":"","mute_first_recovery_notification":false,"notify_end_types":["expired","canceled"],"notify_end_states":["alert","warn","no data"],"schedule":{"timezone":"Europe/Berlin","recurrences":[{"start":"2023-07-10T18:00","duration":"12h","rrule":"FREQ=DAILY;INTERVAL=1"}]}}},"_authentication_token":"b6c9ec89cdff687d29c0ee54923c52f57c9e102a"}'
+```
+
+Optionally, add a `message` to your Downtime to let others know the reason and purpose of the Downtime you are creating. Replace the placeholder value `<DATADOG_SITE>` with the right site parameter of your Datadog account, see [here][1]. Replace the `start` and `end` parameter to match your wanted schedule.
+
+**Response:**
+
+```json
+{
+  "data": {
+    "type": "downtime",
+    "id": "16d09s97-1a70-11ee-8319-dasan1997",
+    "attributes": {
+      "message": "",
+      "mute_first_recovery_notification": false,
+      "canceled": null,
+      "scope": "env:prod",
+      "monitor_identifier": { "monitor_tags": ["*"] },
+      "modified": "2023-07-05T08:12:17.145771+00:00",
+      "created": "2023-07-05T08:12:17.145771+00:00",
+      "status": "scheduled",
+      "display_timezone": "Europe/Berlin",
+      "schedule": {
+        "recurrences": [
+          {
+            "duration": "12h",
+            "rrule": "FREQ=DAILY;INTERVAL=1",
+            "start": "2023-07-10T18:00"
+          }
+        ],
+        "current_downtime": {
+          "end": "2023-07-11T04:00:00+00:00",
+          "start": "2023-07-10T16:00:00+00:00"
+        },
+        "timezone": "Europe/Berlin"
+      },
+      "notify_end_states": ["alert", "warn", "no data"],
+      "notify_end_types": ["canceled", "expired"]
+    },
+    [..]
+  },
+  [..]
+}
+```
+
+[1]: https://docs.datadoghq.com/getting_started/site
+{{% /tab %}}
+{{% tab "UI" %}}
+
+Open the [manage Downtime page][1] and add a new downtime. Select `recurring`:
+
+{{< img src="monitors/guide/downtime_businesshour.png" alt="Downtimes outside of business hours" style="width:100%;" >}}
+
+[1]: https://app.datadoghq.com/monitors#downtime
+{{% /tab %}}
+{{< /tabs >}}
+
+### Combined downtime for outside business hours and weekend
+
+For use cases where you strictly want to be notified by your monitors only during actual business hours, you will need to mute monitors during the week as well as during the weekend. This can be combined in a single Downtime. Continuing the example above:
```

Comment:
```suggestion
For use cases where you only want monitor notifications during business hours, mute monitors during the week as well as during the weekend. This can be combined in a single Downtime. Continuing from the [Downtime outside of business hours](#downtime-outside-of-business-hours) example above:
```